### ISSUE TITLE
Receiving multiple errors on the transaction list screen

### ISSUE THREAD

**Issue by Anziskavds** [tari-project/wallet-ios#459]
When a transaction was sent to my iOS device and the recipient who sends it received the network error and I still received the transaction(where you don't expect to receive the transaction at all) but I also received multiple error messages popping up one after the other. I had the app running in the background on my profile screen and opened the app to view my transaction via the push notification received, it opened on my profile screen and when I clicked back on my profile screen to go back to my transaction list screen I received the multiple error messages. 

**Steps to reproduce the behavior:**
1. On your iOS device open your profile screen 
2. Background your app while you were on the profile screen
3. Receive a transaction from a recipient but the sender should receive the "Network error" message as they are sending the transaction to your wallet
4. The receiver(you) should receive a push notification after the transaction was sent
5. Open the app by clicking on the push notification
6. The app should open on your profile screen
7. Click back from the profile screen to go to your transaction list screen
8. Verify that the transaction reflects on your transition list screen
9. Wait a few seconds to see if you receive multiple error messages popping up on the transaction list screen

**Video of the behavior:**
https://nomatter.tari.com/files/ni6f7iheub8i5rsm5zb83b19hy/public?h=-kRg5Zzhv2B4NSfAXURjKuNHuDZ1wJEq20KmzhDAoIM
 
- Device: iPhone 7
- OS: 13.5.1

**Comment by Tas4tari** [comment]
As per @Jasonvdb this is not a bug and they need to block out those UI errors

**Comment by Anziskavds** [comment]
Trying to test this but I haven't received any network error messages while sending transactions to the iOS. I would only be able to test this when I receive a network error again.

cc @Jasonvdb @Tas4tari

**Comment by Anziskavds** [comment]
Tested by sending a transaction from another device and receiving the network error to my iOS. I received the network error on my Android device after I tried to initiate a transaction to my iOS device. No transaction was received on iOS and also no transaction shows on the transaction list screen on either the Android device or the iOS device. Seems to be tested successfully seeing that the transaction didn't go through to the iOS when I received the network error on the Android. I also haven't received any multiple error pop up messages while testing this.

![Screenshot_20200727-133416_Tari Aurora.jpg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/02d5d41f-1577-4779-ba45-81325aad9965)

![Screenshot_20200727-133440_Tari Aurora.jpg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/9ddfc6cc-c21e-4dc4-a276-51c5ed7fc858)

![Screenshot_20200727-133449_Tari Aurora.jpg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/312c480d-a1c8-4857-bb43-c11b30cc3361)


![Screenshot_20200727-133511_Tari Aurora.jpg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/558ff48b-f28b-4a30-8bb1-cd723b7ee25e)
